third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -316,7 +316,7 @@ addNewScriptHandler(async (scriptId, sourceURL, relativeSourceMapURL) => {
     try {
       sourceContent = await getCachedResource(url, scriptHash);
     } catch (err) {
-      log(`[RuntimeError] Failed to read original source ${url}: ${err.message}`);
+      log(`[RuntimeError][sourcemaps] Failed to read original source ${url}: ${err.message}`);
```

Comment:
worth having `logError`-like function up here (as down on line 359?)

also, why `log` here but `warning` down on 337?

Reply:
> worth having logError-like function up here
I think there are many things that are in dire need of refactoring here. But I'm going for the minimal-effort approach here 😅 

> why log here but warning down on 337?
Most failures in `gSourcemapScript` are just bad sourcemaps, including this one (e.g. a bad url, or a `webpack:///...` url would trigger this `catch`). Since bad sourcemaps are a common problem that we have not yet started to tackle, we don't want to generate too much noise, especially since "warnings" are honeycomb-resident and costly, and might wash out other, more important warnings.

The `warning` below indicates an actual failure in our script. It should never happen.